CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+## Contributing
+Thank you for your interest in contributing to Codewind.
```

Comment:
May want to add a line or two to say something like contributions are very welcome.